### PR TITLE
update querys

### PR DESCRIPTION
**Pull request by HectorTorresE**
In this PR the followings files were modified:

- schema.sql to alter the table and add the column ``species``
- data.sql to add the new 7 animals (For a total of 11 animals)
- queries.sql to use transactions to modify and delete data in the animals' table. All instructions from the list of project requirements were followed.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,17 +1,62 @@
...
+  SAVEPOINT date_filter;
+
+  UPDATE animals SET weight_kg = (weight_kg * -1);
+
+  ROLLBACK TO date_filter;
+
+  UPDATE animals SET weight_kg = (weight_kg * -1) WHERE weight_kg < 0;
+  
+COMMIT TRANSACTION;
+
```
Comment (mahbubul14) [inline queries.sql:46]: - [ ] You should add screenshots of your transaction results to the pull request description. :+1:

File: queries.sql [hunk 2]
```diff
@@ -1,17 +1,62 @@
...
+SELECT CASE 
+WHEN neutered=true THEN 'Neutered' ELSE 'Not Neutered' 
+END,
+SUM(escape_attempts) as "escapes"
+FROM animals GROUP BY neutered ORDER BY escapes DESC LIMIT 1;
+
+  
+SELECT species, MIN(weight_kg) AS "Minimum weight", MAX(weight_kg)  AS "Maximum weight" FROM animals GROUP BY species;
+
+SELECT species, AVG(escape_attempts) FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
```
Comment (mahbubul14) [inline queries.sql:62]: - [ ] You should add a screenshot of your query results to the pull request description. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @HectorTorresE ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by andersonlebon — APPROVED** [review]
## Hi @HectorTorresE 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**